storage/migrations/sqlite3/2_tokens.sql

```diff
@@ -1,15 +1,15 @@
 -- +migrate Up
 
-CREATE TABLE IF NOT EXISTS "tokens" (
-  "id"         integer primary key autoincrement,
-  "user_id"    integer,
-  "token"      varchar(191) NOT NULL UNIQUE,
-  "type"       varchar(191) NOT NULL,
-  "expires_at" datetime NOT NULL,
-  "created_at" datetime NOT NULL,
-  "updated_at" datetime NOT NULL
+CREATE TABLE IF NOT EXISTS `tokens` (
```

Comment:
This is sqlite3 so we'll need to revert this.

Reply:
Yup you are right.